tests/testthat/tests_modelEstimation.R

```diff
@@ -0,0 +1,140 @@
+context("testing modelEstimation")
+
+test_that("modelEstimation defaults works", {
+  skip_on_cran()
+  
+  eList_MO <- Missouri_eList
+  info_orig <- getInfo(eList_MO)
+  daily_orig <- getDaily(eList_MO)
+  sample_orig <- getSample(eList_MO)
+  surfaces_orig <- getSurfaces(eList_MO)
+  
+  eList_MO_modeled <- modelEstimation(eList_MO)
+  info <- getInfo(eList_MO_modeled)
+  daily <- getDaily(eList_MO_modeled)
+  sample <- getSample(eList_MO_modeled)
+  surfaces <- getSurfaces(eList_MO_modeled)
+  
+  ## INFO ##
+  
+  # info retained columns are unaltered after running modelEstimation
+  end_orig_cols_info <- which(names(info) == "bottomLogQ") - 1
+  expect_true(all(names(info_orig) == names(info[,1:end_orig_cols_info])))
+  expect_equal(sapply(info_orig, '[[', 1), sapply(info[,1:end_orig_cols_info], '[[', 1))
+  
+  # defaults show up in INFO
+  expect_equal(info[['windowY']], 7)
+  expect_equal(info[['windowQ']], 2)
+  expect_equal(info[['windowS']], 0.5)
+  
+  ## DAILY ##
+  
+  # daily data is unaltered after running modelEstimation
+  expect_equal(mean(daily[['Date']]), mean(daily_orig[['Date']]))
```

Comment:
Ah good point, I think means popped in my head and I didn't consider any other alternative. Will change that for the data the appears in the original dataset.